cli/.pylintrc

```diff
@@ -195,7 +197,7 @@ score=yes
 [REFACTORING]
 
 # Maximum number of nested blocks for function / method body
-max-nested-blocks=5
+max-nested-blocks=6
```

Comment:
I'd avoid extending this, we can instead define a function/method for some of the nested blocks.

Reply:
Done